sdk/python/requirements.txt

```diff
@@ -132,8 +132,10 @@ termcolor==1.1.0
     # via fire
 typer==0.4.1
     # via -r requirements.in
-typing-extensions==4.2.0
-    # via pydantic
+typing-extensions==4.2.0 ; python_version < "3.9"
+    # via
+    #   -r requirements.in
+    #   pydantic
```

Comment:
It looks like the `requirements.txt` was not affected by your particular change and that this is attributed to the version of Python.

Can you revert this change?